### ISSUE TITLE
Remove vertical scroll from main map container

### ISSUE THREAD

**Issue by bencpeters** [oviohub/prism-frontend#62]
Right now there's a slight vertical scroll for the main map container. This is because the map view is sized using `100vh`, which doesn't account for the height of the navbar.

We should redo the structure of the page to make the the main wrapper a column-oriented flex container, with the navbar taking up its fixed height and the `main` container growing to fill the rest of the available space.

**Comment by ericboucher** [comment]
Actually I also have a horizontal one

**Comment by ericboucher** [comment]
Seems fixed?

**Comment by bencpeters** [comment]
> Seems fixed?

Nope. It's just easy to not see it since the map takes over the scroll for zooming if your mouse is over it. Put your mouse over the navbar and scroll down to see it.

**Comment by ericboucher** [comment]
Indeed!

**Comment by SimplyBallistic** [comment]
I'll take a shot at this after #54 is closed! 

I also noticed a slight horizontal scroll as well. I'll try fix both in my PR.

**Comment by bencpeters** [comment]
> I'll take a shot at this after #54 is closed!
> 
> I also noticed a slight horizontal overflow as well. I'll try fix both in my PR.

I actually have some code for this already @SimplyBallistic, let's make it part of a separate PR to keep concerns separated.

**Comment by bencpeters** [comment]
I just added a PR with the code I had on a previous branch: #79. Feel free to look at that and make sure it works on your system too?
